floris/simulation/farm.py

```diff
@@ -50,6 +55,9 @@ class Farm(BaseClass):
     layout_x: NDArrayFloat = field(converter=floris_array_converter)
     layout_y: NDArrayFloat = field(converter=floris_array_converter)
     turbine_type: List = field()
+    turbine_library: Path = field(
```

Comment:
It might be more accurate to call this attribute `turbine_library_path`. In reading the unit test you added, it seems odd to compare `farm.turbine_library` to a string.

Reply:
Happy to change, but I'm not sure which part of the test you're referring to. It seems like the external_library comparison and the default_library are both `Path` objects, or they'd fail.

Reply:
Oh woops - I misread the unit test.
What I meant to get at here is that `turbine_library` sounds like it is the library itself. Without knowing anything about it, I might try to pull a turbine out of it. However, it would be more explicit to indicate that its actually the filepath to the turbine definitions.